app/javascript/packs/document-capture-welcome.ts

```diff
@@ -1,67 +1,9 @@
-import { trackEvent } from '@18f/identity-analytics';
-import { hasCamera, isCameraCapableMobile } from '@18f/identity-device';
```

Comment:
I think this was the only place `hasCamera` was used, so we could remove it now as well.

Reply:
Right, was thinking we could remove [the original implementation in `device/index.js`](https://github.com/18F/identity-idp/blob/85cb170e4b1bf4392afc768f3a6840a5516245f7/app/javascript/packages/device/index.js#L33-L43) since it's now unused.

Reply:
Thanks, for some reason that didn't come up when I searched, but it's removed now.